server/sonar-web/src/main/js/apps/grc/create-Project/CreateProjectPage.tsx

```diff
@@ -115,7 +115,7 @@ export default class CreateProjectPage extends React.PureComponent<Props & WithR
   };
 
   handleClose = () => {
-    this.props.router.replace('/grc');
+    this.props.router.replace('/grc/dashboard');

```

Comment:
this requires a id in the url

Reply:
I will fix this in next PR. Need some more clarity here. For now let it be as is.